swagger-templates/csharp/ApiClient.mustache

```diff
@@ -112,7 +112,16 @@ namespace {{packageName}}.Client
 
             // add header parameter, if any
             foreach(var param in headerParams)
-                request.AddHeader(param.Key, param.Value);
+            {
+                if (param.Key.Equals("Authorization"))
```

Comment:
Hmm, I'm not that familiar with C#, but I wonder if it's more conventional to use `string1 == string2`, or `string1.Equals(string2)`?